slsc-face-archive.md

```diff
@@ -0,0 +1,15 @@
+# Project SLSC Photo Archive Face Tagging
+
+## Client Profile
+
+Youth Exploring Science (YES) program is created to help underserved students in St. Louis area. Students participate in YES program to gain skills in Science, Technology, Engineering, Arts and Mathematics fields. High school students will gain real-world skills, career readiness training through YES program. Students in the YES program also promote the importance of STEAM-related careers.
+
+## Project Brief
+
+The SLSC Face Archive Project captures and preserves various facial expressions and emotions, assisting researchers, psychologists, and developers in disciplines such as mental health, AI, and emotion identification. It adheres to ethical norms, ensuring participant consent, privacy, and anonymity, and has a wide range of applications, including investigating emotional responses and refining AI algorithms.
```

Comment:
The SLSC Photo Archive Face project is to Identify and Tag people from the images client or user uploads, next once the detected faces tagging is completed we need to store the organized result in a database readily available to access.

Reply:
incorporated comments into updates.